tests/phpunit/integration/Core/Admin/ScreenTest.php

```diff
@@ -103,7 +103,6 @@ public function test_initialize() {
 
 	public function test_enqueue_assets() {
 		wp_dequeue_style( 'googlesitekit_admin_css' );
-		wp_dequeue_script( 'googlesitekit_modules' );
```

Comment:
Instead of removing this, let's exchange it with another script that we still have, to make sure enqueuing scripts is still covered.

Reply:
I did this in [`AssetsTest`](https://github.com/google/site-kit-wp/pull/1013/files#diff-3dff40ce7dc41bb9397872c63bf36a96R102) since this test is more specific to assets enqueued globally across many screens. Of course adding back a different asset would not be a problem though. Apart from fonts, `googlesitekit_admin_css` is the only asset enqueued on all screens.
https://github.com/google/site-kit-wp/blob/b37b4abf8afb11f61b3727b5206064b61ab9ee4e/includes/Core/Admin/Screen.php#L177-L192

Reply:
SGTM